src/components/ebay-toggle-button-group/toggle-button-group.stories.ts

```diff
@@ -85,41 +108,22 @@ Default.parameters = {
     },
 };
 
-export const WithIcons = (args) => ({
-    input: args,
-    component: withIconsTemplate,
-});
-WithIcons.args = {};
-WithIcons.parameters = {
-    docs: {
-        source: {
-            code: withIconsCode,
-        },
-    },
-};
+export const WithIcons = buildExtensionTemplate(
```

Comment:
Even with the columns being null, we're still setting them in the markup. When those params are not included, those attributes should not even be there. 

<kbd><img width="1402" alt="image" src="https://github.com/eBay/ebayui-core/assets/1675667/2fa6fb9b-4549-4498-a4c4-8dcace5b6d1d"></kbd>

Reply:
We can remove them from examples. However, if teams use those examples to see what thigns should be like then they won't see them. 
They are not being set by default.